.ci/jobs/elastic+eui+pull-request-test.yml

```
@@ -11,7 +11,7 @@
           #!/usr/local/bin/runbld
           export NVM_DIR="/var/lib/jenkins/.nvm"
           [[ -s "$NVM_DIR/nvm.sh" ]] && . "$NVM_DIR/nvm.sh"
-          NPM_VERSION=10
+          NPM_VERSION=16
           nvm install ${NPM_VERSION}
           nvm use ${NPM_VERSION} --delete-prefix
```

Making a suggestion for this (see above comment) and specifying the NODE_VERSION to match our docker images, to make it easier to grep for a specific version to change simultaneously in the future
```suggestion
          NODE_VERSION=16.13.1
          nvm install ${NODE_VERSION}
          nvm use ${NODE_VERSION} --delete-prefix
```